src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,76 @@
-export const Pagination = () => {};
+import React from 'react';
+
+interface PaginationProps {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<PaginationProps> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+
+  const handlePageChange = (page: number) => {
+    if (page >= 1 && page <= totalPages) {
+      onPageChange(page);
+    }
+  };
+
+  return (
+    <ul className="pagination">
+      <li className={`page-item ${currentPage === 1 ? 'disabled' : ''}`}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          onClick={(event) => {
+            event.preventDefault();
+            handlePageChange(currentPage - 1);
+          }}
+          aria-disabled={currentPage === 1}
+        >
+          «
+        </a>
+      </li>
+
+      {Array.from({ length: totalPages }, (_, i) => i + 1).map((page) => (
+        <li
+          key={page}
+          className={`page-item ${currentPage === page ? 'active' : ''}`}
+        >
+          <a
+            data-cy="pageLink"
+            className="page-link"
+            href={`#${page}`}
+            onClick={(event) => {
+              event.preventDefault();
+              handlePageChange(page);
+            }}
+          >
+            {page}
+          </a>
+        </li>
+      ))}
+
+      <li className={`page-item ${currentPage === totalPages ? 'disabled' : ''}`}>
+        <a
+          data-cy="nextLink"
+          className="page-link"
+          href="#next"
```

Comment:
Similar to the previous comment about `href="#prev"`, using `href="#next"` can cause the page to jump to the top. Consider using `href="javascript:void(0);"` or a button instead of a link.